zkevm-circuits/src/evm_circuit.rs

```diff
@@ -475,4 +484,81 @@ mod evm_circuit_stats {
             |_, state, _| state.get_step_height_option().unwrap(),
         );
     }
+
+    /// This function prints to stdout a table with the top X ExecutionState
+    /// cell consumers of each EVM Cell type.
+    ///
+    /// Run with:
+    /// `cargo test -p zkevm-circuits --release get_exec_steps_occupancy
+    /// --features test -- --nocapture --ignored`
+    #[ignore]
+    #[test]
```

Comment:
What do you think we move it out from tests and run it with the `cargo run` command? This can simplify the command.

Reply:
That would require to move this to a binary tool. I discussed this with @ed255 and I think we can for example, file a `tech-debt` issue which moves all of the stats-extracting tests into a cli tool or similar. 

Would that work?

Reply:
Yes, I think filing an issue and working on it later works.